llvm/lib/Target/WebAssembly/WebAssemblyInstrSIMD.td

```diff
@@ -702,6 +702,19 @@ defm "" : ReplaceLane<I64x2, 30>;
 defm "" : ReplaceLane<F32x4, 32>;
 defm "" : ReplaceLane<F64x2, 34>;
 
+// For now use an instrinsic for f16x8.replace_lane instead of ReplaceLane above
+// since LL generated with half type arguments is not well supported and creates
```

Comment:
What is LL? I may lack the context because I haven't reviewed your other recent PRs..

Reply:
I think this just refers to LLVM IR (but I agree that using that term is more consistent with what I've seen elsewhere).

Reply:
Yeah, I'll update.